### ISSUE TITLE
Snapping

### ISSUE THREAD

**Issue by ttencate** [ttencate/taekwindow#10]
Converted from [SourceForge issue 1629765](http://sourceforge.net/support/tracker.php?aid=1629765), submitted by ttencate

Windows snapping to screen edges and to each other while moving and resizing.

**Comment by ttencate** [comment]
Submitted by ttencate

Logged In: NO 

Would really love this feature...
Or just make it work with 'allSnap' or a similar app, please please please!

**Comment by ttencate** [comment]
Submitted by ttencate

I second this!

Working with allSnap or integrating similar functionality is key.

**Comment by diego10arg** [comment]
I agree with this. AllSnapX86 + AllSnapX64 + TaekWindow + Winroll = Perfect Combo!
